tests/routes/admin_login.test.js

```diff
@@ -0,0 +1,51 @@
+const test = require('tape');
+const request = require('supertest');
+const jwt = require('jsonwebtoken');
+const createApp = require('../../react-backend/app');
+const { getConfig } = require('../../config');
+
+const config = getConfig(process.env.NODE_ENV);
+
+test('POST /api/admin/login | password match database hash', t => {
+  const app = createApp(config);
+  const dbConnection = app.get('client:psql');
+  const secret = app.get('cfg').session.standard_jwt_secret;
+
+  const token = jwt.sign({ email: 'jinglis12@googlemail.com' }, secret);
+
+  const successPayload = {
+    password: 'Sallydog7&',
+  };
+  request(app)
+    .post('/api/admin/login')
+    .set('authorization', token)
+    .send(successPayload)
+    .expect(200)
+    .expect('Content-Type', /json/)
+    .end((err, res) => {
+      t.equal(res.body.success, true);
```

Comment:
Here you're not checking if supertest is erroring on any of the `.expect`'s. You can check this by adding in the line:

`t.notOk(err, err || 'Passes supertest expect criteria')`

This checks that error is undefined, if it's not the test fails 😊